docs/iris/src/userguide/saving_iris_cubes.rst

```diff
@@ -54,6 +56,16 @@ The :py:func:`iris.save` function passes all other keywords through to the saver
     >>> # Save a cube list to netCDF, using the NETCDF3_CLASSIC storage option
     >>> iris.save(cubes, "myfile.nc", netcdf_format="NETCDF3_CLASSIC")
 
+.. testcleanup::
+
+    import pathlib
+    p = pathlib.Path("myfile.pp")
+    if p.exists():
+        p.unlink()
+    p = pathlib.Path("myfile.nc")
+    if p.exists():
+        p.unlink()
+
```

Comment:
I've been meaning to fix this for years... it simply tidies up the `myfile.pp` and `myfile.nc` created by the `doctest`.

Hence there will be less `unstaged` files in your branch after running the `doctests`. Win.